scripts/tag-release-candidate.sh

```diff
@@ -79,9 +80,10 @@ if [ -z "$CS_VERSION" ]; then
   exit 1
 fi
 
-# If a user omits the leading "v" (enters 1.2.3 instead of v1.2.3), handle it.
-if ! [[ "${CS_VERSION}" = v* ]]; then
-  CS_VERSION="v${CS_VERSION}"
+# If a user omits the leading "v" (enters 1.2.3 instead of v1.2.3), throw an error.
+if ! [[ "${CS_VERSION}" =~ ^v[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+  echo "Error: CS_VERSION must be in the form v1.2.3 - got ${CS_VERSION}"
```

Comment:
Combine this with L78-81 if we choose to throw an error here

Reply:
I think having separate conditions and associated error messages is valid, it makes it clear whether the error was that
- The `CS_VERSION` was not provided
- The `CS_VERSION` was provided but is in an incorrect format

WDYT?

Reply:
I removed L78-81